core/src/config/sequelize.ts

```diff
@@ -2,7 +2,7 @@ import fs from "fs";
 import { URL } from "url";
 import { join, isAbsolute } from "path";
 import { getParentPath, getPluginManifest } from "../modules/pluginDetails";
-import { log } from "actionhero";
+import { log, env } from "actionhero";
```

Comment:
... It might be that maybe using `{env}` was a bad idea - it might not be ready yet when the config is being calculated :/

Reply:
```
npm run dev

> @grouparoo/app-staging-enterprise@0.7.0-alpha.10 dev
> cd node_modules/@grouparoo/core && ./bin/dev

[nodemon] 2.0.12
[nodemon] to restart at any time, enter `rs`
[nodemon] watching path(s): src/**/* ../plugins/**/*.js
[nodemon] watching extensions: js,jsx,ts,tsx
[nodemon] starting `node_modules/.bin/ts-node --transpile-only --log-error src/grouparoo.ts`
{ env: undefined }
{ env: undefined }
{ env: undefined }
{ env: undefined }
{ env: undefined }
2021-10-04T22:44:55.905Z - notice: Starting Grouparoo v0.7.0-alpha.10 on node.js v16.6.1
{ env: 'development' }
{ env: 'development' }
```